### DISCUSSION TITLE
리뷰 작성 기능 추가 제안

### DISCUSSION THREAD

**Discussion by shinseongsu** [9in-team/Backend_java#16] [Ideas]
프로젝트가 끝난후, 리뷰를 작성하는 것이 있었으면 좋겠어요~

**Comment by shinseongsu** [comment]
내 지원 목록에 상태는 대기중, 초대됨, 거절됨 ,종료 같은 상태값이 있었으면 좋곘어요

- 대기중 : 프로젝트 지원을 한 상태
- 초대됨 : 프로젝트 지원을 팀장이 허락한상태
- 거절됨 : 팀장이 지원자를 거절한 상태
- 종료 : 프로젝트가 종료가 선언되었을때 상태

https://ovenapp.io/view/WWHNr8bmgjLdTUOFDSt6fAU5phOXqfd7/Z8t49

**Comment by shinseongsu** [comment]
종료 된 지원서를 누르게 될 경우 팀원 리뷰 목록으로 갈 수 있습니다.

https://ovenapp.io/view/WWHNr8bmgjLdTUOFDSt6fAU5phOXqfd7/5Ulyj

**Comment by shinseongsu** [comment]
한사람당 리뷰를 사용할 수 있습니다.
- 별로예요 : 평판점수를 하락 시킵니다.
- 좋아요: 평판점수를 상승시킵니다.
- 최고예요: 평판점수를 많이 상승시킵니다. ( 구체적인 방안 합의 예정 )

https://ovenapp.io/view/WWHNr8bmgjLdTUOFDSt6fAU5phOXqfd7/kLLkt

**Reply by t1dmlgus** [reply]
해당 내용 관련해서 리뷰서비스 구성해봤습니다.
아래 이슈 확인 부탁드립니다:)
- https://github.com/9in-team/Backend_java/issues/20

**Comment by shinseongsu** [comment]
리뷰 후, FCM 메시지로 리뷰한 사람에게 익명으로 메시지를 보낼 수 있다. ( 선택 )

https://ovenapp.io/view/WWHNr8bmgjLdTUOFDSt6fAU5phOXqfd7/fm73L

https://ovenapp.io/view/WWHNr8bmgjLdTUOFDSt6fAU5phOXqfd7/LExWI
